src/components/TokenSelector.tsx

```diff
@@ -252,6 +252,32 @@ interface Props {
 const TokenSelector: React.FC<Props> = ({ isDisabled, tokens, selected, onChange, tabIndex = 0 }) => {
   const options = useMemo(() => tokens.map(token => ({ token, value: token.symbol, label: token.name })), [tokens])
 
+  // isFocused is used to force the menu to remain open and give focus to the search input
+  const [isFocused, setIsFocused] = useState(false)
```

Comment:
So the answer is to force our way through? :laughing:

Reply:
YES! We are taking "manual control" over the select when this is set.